canonical-websites/www.ubuntu.com
# Updated Ubuntu Advantage service description

## Done

- Updated Ubuntu Advantage service description
- Made Appendix 1 an include
- Used includes from the BootStack service description
- Fixed some of the css issues for font size of number bullets and indents of paragraphs, plain lists and tables

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [service description](http://0.0.0.0:8001//legal/ubuntu-advantage/service-description)
- compare to the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#)
- ask @chrisjohnston about the position of the ‘Severity levels and target response times’ include

## Issue / Card

Fixes #2748 and partially fixes #2749
